javalib/src/main/scala/java/security/Guard.scala

```diff
@@ -0,0 +1,5 @@
+package java.security
+
+trait Guard {
+  def checkGuard(o: Any): Unit
```

Comment:
We usually use ```Any```
https://docs.oracle.com/javase/7/docs/api/java/util/AbstractCollection.html#contains(java.lang.Object)
https://github.com/scala-js/scala-js/blob/master/javalib/src/main/scala/java/util/AbstractCollection.scala#L14